domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/error/StopTooFarFromTripShape.java

```diff
@@ -0,0 +1,32 @@
+package org.mobilitydata.gtfsvalidator.domain.entity.notice.error;
+
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.NoticeExporter;
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.base.ErrorNotice;
+
+import java.io.IOException;
+
+public class StopTooFarFromTripShape extends ErrorNotice {
+
+    public StopTooFarFromTripShape(final String filename, final String stopId, final int stopSequence,
+                                   final String tripId, final String shapeId, final double stopShapeThreshold) {
+        super(filename, E_052,
```

Comment:
in `GeospatialUtilsImpl` we use this method with filename=`shapes.txt`: we could remove this field, and use a static String directly in call to super method. 
Also, since the problematic file is `shapes.txt`, I think that entity id should be set to `shapeId`.

Should we create new static key for this notice? Since `shapes.txt` has a defined `id` field, there is no need to define composite key values, we could have keys such as `KEY_STOP_STOP_ID`, `KEY_TRIP_TRIP_ID`, `KEY_STOP_STOP_SEQUENCE`. The linked information would be the same, but I think that it improves the code consistency and legibility for the future.

Reply:
>in GeospatialUtilsImpl we use this method with filename=shapes.txt: we could remove this field, and use a static String directly in call to super method.

👍  Will change this

>Also, since the problematic file is shapes.txt, I think that entity id should be set to shapeId.
>
>Should we create new static key for this notice? Since shapes.txt has a defined id field, there is no need to define composite key values, we could have keys such as KEY_STOP_STOP_ID, KEY_TRIP_TRIP_ID, KEY_STOP_STOP_SEQUENCE. The linked information would be the same, but I think that it improves the code consistency and legibility for the future.

It's possible the problem that this rule detects is actually a stop in a wrong location, not the shape being wrong. So I think it's best to keep the composite key, as the individual error is a combination of the shape, stop sequence, and stop ID.